bip-0360.mediawiki

```diff
@@ -11,696 +11,386 @@
-
-It should be noted that Taproot outputs are vulnerable in that they encode a 32-byte x-only public key, from which a
-full public key can be reconstructed.
-
-If a CRQC recovers an extended public key (xpub), including its chain code, it can derive all non-hardened child public
-keys by guessing or iterating through child indexes, as allowed by BIP 32's non-hardened derivation. With Shor's
-algorithm, the CRQC could then compute the corresponding non-hardened child private keys directly from those public keys,
-without needing the extended private key (xprv) or an exposed child private key. Hardened child keys remain secure since
-they cannot be derived from the xpub alone. However, if the xprv is exposed, then all child private keys--both hardened
-and non-hardened--become vulnerable. Thus, in a quantum context, the xpub alone is sufficient to expose all non-hardened
-child private keys.
+A complete list of outputs that are vulnerable to long-exposure attacks:
 
-==== Long Exposure and Short Exposure Quantum Attacks ====
+* Legacy (i.e. P2PK) outputs (Satoshi’s coins, CPU miners)
+* Reused outputs of any type
+* Taproot outputs (mainnet start with bc1p)
 
-A Long Exposure Quantum Attack is an attack in which the public key has been exposed on the blockchain for an extended
-period of time, giving an attacker ample opportunity to break the cryptography. This affects:
+For further clarification on quantum attack vectors, please refer to our [[#Glossary|Glossary of Terms]].
 
-* P2PK outputs (Satoshi's coins, CPU miners, starts with 04)
-* Reused addresses (any type, except P2QRH)
-* Taproot addresses (starts with bc1p)
-* Extended public keys, commonly known as "xpubs"
-* Wallet descriptors
+==Design==
 
-A Short Exposure Quantum Attack is an attack that must be executed quickly while a transaction is still in the mempool,
-before it is mined into a block. This affects:
+Pay-to-Tapscript-Hash (P2TSH) is a Tapscript-native, hashed output type that commits solely to the root of a Tapscript tree. Functionally, it is a modified Pay-to-Taproot (P2TR) output with the quantum-vulnerable key-path spend disabled by omission of the internal keys and of the Taproot 'tweak' step.
 
-* Any transaction in the mempool (except for P2QRH)
+In other words, P2TSH outputs commit to the Merkle root of a Tapscript tree without committing to any internal public key. The script(s) being committed to, however, may contain keys or key-hashes.
 
-Short-exposure attacks require much larger, more expensive CRQCs since they must be executed within the short window
-before a transaction is mined. Long-exposure attacks can be executed over a longer timeframe since the public key remains
-exposed on the blockchain indefinitely.
+This output type is designed to offer Taproot users immediate protection against long-exposure quantum attacks – as well as to provide a practical output type with which new, post-quantum signatures may evolve for additional security.
```

Comment:
```suggestion
This output type is designed to offer taproot users protection against long-exposure quantum attacks -- as well as a practical output type with which post-quantum signatures may be used for additional security.

Since P2TSH has no key path spend, P2TSH omits the Taproot internal key. Instead, a P2TSH output includes only the 32-byte root of the tapleaf Merkle tree as defined in [https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki BIP 341] hashed with the tag "MerkleRoot" as shown below.
```

Reply:
Notice there was a paragraph missing

Reply:
@cryptoquick The Word document provided contained incorrect specification details. We discussed this, it was explained to you how this actually works in Taproot, and you OK'd the changes.